globals.go

```diff
@@ -41,18 +41,26 @@ const (
 
 	HostExternalUserName = "external"
 
-	// Task Statuses used in the database models
+	// Task Statuses used in the database models or in the UI
 
-	// TaskInactive is not assigned to any new tasks, but can be found
-	// in the database and is used in the UI.
+	// TaskInactive is a deprecated legacy status that used to mean that the
+	// task was not scheduled to run. This is equivalent to the TaskUnscheduled
+	// display status. These are not stored in the task status (although they
+	// used to be for very old tasks) and may be still used in some outdated
+	// pieces of code.
 	TaskInactive = "inactive"
 
 	// TaskUndispatched indicates either:
 	//  1. a task is not scheduled to run (when Task.Activated == false)
 	//  2. a task is scheduled to run (when Task.Activated == true)
 	TaskUndispatched = "undispatched"
-	TaskUnscheduled  = "unscheduled"
-	// TaskWillRun is a subset of undispatched tasks and is only used in the UI.
+	// TaskUnscheduled indicates that the task is undispatched and is not
+	// scheduled to eventually run. This is a display status, so it's only used
+	// in the UI.
+	TaskUnscheduled = "unscheduled"
```

Comment:
Can we similarly organize all display statuses into a block?

Reply:
Done.